Sources/Security/Signing+ResponseVerification.swift

```diff
@@ -92,17 +95,10 @@ extension HTTPResponse where Body == Data {
             return .failed
         }
 
-        let eTag = HTTPResponse.value(forCaseInsensitiveHeaderField: HTTPClient.ResponseHeader.eTag.rawValue,
-                                      in: headers)
-
-        let messageToSign = statusCode == .notModified
-            ? eTag?.asData ?? .init()
-            : body
-
         if signing.verify(signature: signature,
                           with: .init(
-                            message: messageToSign,
-                            nonce: nonce,
+                            message: body,
```

Comment:
So this will be true even if the body is empty right? (in the case of 304 responses I'm thinking)

Reply:
Ah ignore me that was in the next PR 😅